src/createServer.js

```diff
@@ -1,3 +1,98 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('node:http');
+const { convertToCase } = require('./convertToCase');
+
+const OK = 200;
+const BAD_REQUEST = 400;
+
+const CASE_OPTIONS = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+/* eslint-disable max-len */
+const NO_TEXT_MESSAGE =
+  'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+
+const NO_TO_CASE_PARAM_MESSAGE =
+  '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+
+const TARGET_CASE_NOT_SUPPORTED = `This case is not supported. Available cases: ${CASE_OPTIONS.join(', ')}.`;
+/* eslint-enable max-len */
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const [path, query] = req.url.split('?');
+    const originalText = path.split('/')[1];
+    const targetCase = new URLSearchParams(query).get('toCase');
+
+    const errors = [
+      ...evalTextErrors(originalText),
+      ...evalToCaseErrors(targetCase),
```

Comment:
The use of `evalTextErrors` and `evalToCaseErrors` functions is correct here, but ensure that these functions are properly handling all possible error cases related to text and case conversion. If there are additional error conditions, consider adding them to these functions.